compile_state/Cargo.toml

```diff
@@ -7,3 +7,4 @@ edition = "2021"
 
 [dependencies]
 slvm = { path = "../vm" }
+bridge_types = { workspace = true }
```

Comment:
This dep does not seem to be required for compile_state.  Remove unless I am missing something...

Reply:
removed.